### DISCUSSION TITLE
[SCRAPER] - support request for asian cooking site xiachufang.com

### DISCUSSION THREAD

**Discussion by MaoZ1993** [mealie-recipes/mealie#4003] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

www.xiachufang.com is the best recipe website for Chinese-Asian cooking. It's very well organised and easy to use. All recipes come with a detailed image and instructions.

The quality of the recipes is excellent, and many chefs share their professional-level recipes on it. It would be extremely useful if the scraper and batch import could support this site, as it would fill a gap in the Asian cooking market.

### Please provide a concise description of the feature that would resolve your issue.

Add support for xiachufang.com. This well-organised site is a perfect target for a scraper.
Sample URL:
https://www.xiachufang.com/recipe/103708953/
https://www.xiachufang.com/recipe/106932538/
https://www.xiachufang.com/recipe/103327266/
https://www.xiachufang.com/recipe/107350861/
https://www.xiachufang.com/recipe/106062946/
https://www.xiachufang.com/recipe/104477146/

### Please consider and list out some caveats or tradeoffs made in your design decision

N/A

I'm willing to help to testing

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [X] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Sorosliu1029** [comment]
mealie's scraper is based on [recipe-scrapers](https://github.com/hhursev/recipe-scrapers).
So you can track this issue https://github.com/hhursev/recipe-scrapers/issues/1609 for further support.
